### PR TITLE
fix(FX-4698): search feedback

### PR DESCRIPTION
**Pull request by dimatretyak**
The type of this PR is: **Fix**

This PR solves [FX-4698]

Review app: [send-search-feedback](http://send-search-feedback.artsy.net/)

#### Before
https://user-images.githubusercontent.com/3513494/229781526-2a4a48c7-79e5-4d88-ae06-366f78032692.mp4

#### After
https://user-images.githubusercontent.com/3513494/229781895-5e58364f-97ee-42de-a3da-f5ff2cf9d829.mp4

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[FX-4698]: https://artsyproduct.atlassian.net/browse/FX-4698?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/Search/Components/__tests__/SendFeedback.jest.tsx [hunk 1]
```diff
@@ -25,7 +26,7 @@ describe("SendFeedback", () => {
     ).toBeInTheDocument()
   })
 
-  it("submits the form", async () => {
+  it("submits the form (not logged-id user)", async () => {
```
Comment (anandaroop) [inline src/Apps/Search/Components/__tests__/SendFeedback.jest.tsx:29]:
```suggestion
  it("submits the form (not logged-in user)", async () => {
```

File: src/Apps/Search/Components/__tests__/SendFeedback.jest.tsx [hunk 2]
```diff
@@ -51,4 +52,32 @@ describe("SendFeedback", () => {
       },
     })
   })
+
+  it("submits the form (logged-id user)", async () => {
```
Comment (anandaroop) [inline src/Apps/Search/Components/__tests__/SendFeedback.jest.tsx:56]:
```suggestion
  it("submits the form (logged-in user)", async () => {
```

**Review by anandaroop — DISMISSED** [review]
🙏🏽 

Did we address [the question](https://artsy.slack.com/archives/C9SATFLUU/p1680614435276849?thread_ts=1680606879.474159&cid=C9SATFLUU) about `url` ?

File: src/Apps/Search/Components/__tests__/SendFeedback.jest.tsx (hunk 1 above)
Comment (dimatretyak) [reply src/Apps/Search/Components/__tests__/SendFeedback.jest.tsx:29]: oh, I made a typo again 😩. thank you for noticing!

**Comment by dimatretyak** [comment]
> Did we address [the question](https://artsy.slack.com/archives/C9SATFLUU/p1680614435276849?thread_ts=1680606879.474159&cid=C9SATFLUU) about `url` ?

added here: https://github.com/artsy/force/pull/12201/commits/c7bffc7962323e7eabc3e3960794aec58db0f055

File: src/Apps/Search/Components/__tests__/SendFeedback.jest.tsx (hunk 1 above)
Comment (dimatretyak) [reply src/Apps/Search/Components/__tests__/SendFeedback.jest.tsx]: corrected

**Review by anandaroop — APPROVED** [review]

**Pull request merged**